fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -427,6 +430,23 @@ public static TStorageFormat analyzeStorageFormat(Map<String, String> properties
         }
     }
 
+    // analyzeCompressionType will parse the compression type from properties
+    public static TCompressionType analyzeCompressionType(Map<String, String> properties) throws AnalysisException {
+        String compressionType;
+        if (properties != null && properties.containsKey(PROPERTIES_COMPRESSION)) {
+            compressionType = properties.get(PROPERTIES_COMPRESSION);
```

Comment:
Does it necessary to remove the PROPERTIES_COMPRESSION from properties?

Reply:
In PropertyAnalyzer.java, any other attributes in properties will be remove after analyzing. I think we should be consistent with the other attributes.